### ISSUE TITLE
Remove wxWidgets from audio code

### ISSUE THREAD

**Issue by generic-pers0n** [saucedacity/saucedacity#30]
Pursuant to moving to Qt, it is a good idea to make our audio code toolkit neutral. Some of this work has already started in `AudioIO` and `RealtimeEffectManager` at the time of initially writing this issue.

This issue isn't just pursuant to moving to Qt, however. This is also pursuant to moving Saucedacity's audio processing code into a separate library, perhaps called "SauceAudio" or something. SauceAudio should not have any dependencies on wxWidgets whatsoever.

Finally, in regards to Saucedacity 1.2: some parts might be complete before Saucedacity 1.2 stable is released, but not as a whole. On the other hand, SauceAudio will not make it for Saucedacity 1.2. It is currently planned that work be started on SauceAudio for Saucedacity 1.3 development.

**Comment by FrostKnight** [comment]
I agree with this wisdom, in many ways.  Being toolkit neutral sounds very awesome... :)

I am not sure which toolkit would be best, for minimalism and still have gui and be stable, but either way, so far so good!
